src/graphs/GraphLoader.jsx

```diff
@@ -0,0 +1,60 @@
+/** eslint verified */
+import React from 'react';
+import PropTypes from 'prop-types';
+import SmallBarStackGraph from './SmallBarStackGraph';
+
+const GraphLoader = (
+  {
+    graphType, data, graphTitle, labelY, width, colors,
+    units,
+    handlerInfoGraph,
+    graphDescription,
+    openInfoGraph,
+  },
+) => (
+  <div>
+    {
+      (graphType === 'SmallBarStackGraph') ? (
+        <SmallBarStackGraph
+          dataJSON={data}
+          colors={colors}
+          graphTitle={graphTitle}
+          labelY={labelY}
+          width={width}
+          height="150"
+          units={units}
+          openInfoGraph={openInfoGraph}
+          handlerInfoGraph={handlerInfoGraph}
+          graphDescription={graphDescription}
+        />
+      ) : ('')
```

Comment:
See multi line ternary comment in Summary.jsx

Reply:
Fixed